src/mgr.sol

```diff
@@ -90,8 +88,11 @@ contract TinlakeManager {
         _;
     }
 
-    modifier operatorOnly {
-        require(msg.sender == operator, "TinlakeMgr/operator-only");
+    // Maker Governance is authorized call tell, unwind, and migrate.
+    address public governance;
+
```

Comment:
I would make this multiuser as well. Can use the `bud` + `kiss` + `diss` https://github.com/livnev/rwa-example/blob/master/src/RwaConduit.sol#L93